.github/workflows/cgc.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy to CGC
+on:
+  release:
+    types:
+      - released
+      - edited
+jobs:
+  cgc:
+    runs-on: ubuntu-20.04
+    name: Deploy
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set the docker tag from Repo Tag
+        id: set_dockertag
+        env:
+          IMAGE: cgc-images.sbgenomics.com/stjude/netbid
+          VERSION_TAG: ${{ github.event.release.tag_name }}
+        run: |
+          jq --arg image "${{ env.IMAGE }}:${{ env.VERSION_TAG }}" '(.requirements | .[] | select(.class == ("DockerRequirement")) | .dockerPull) |= $image' cgc/netbid.cwl > cgc/netbid.cwl.new
+          mv cgc/netbid.cwl.new cgc/netbid.cwl
+          cat cgc/netbid.cwl
```

Comment:
@adthrasher Ran this locally via `act` and it looks to work okay.  let me know what you think

Reply:
I think that looks good.